### ISSUE TITLE
Invalid input to URL constructor

### ISSUE THREAD

**Issue by turkeylurkey** [OpenLiberty/liberty-tools-intellij#631]
Found in the IntelliJ output during testing. Need to investigate how to turn a file into a url.
```
2024-01-30 06:48:23,160 [2351960]   WARN - #io.openliberty.tools.intellij.runConfiguration.LibertyRunManagerListener - Unable to clear custom start parameters for Liberty run configuration associated with: /home/runner/work/liberty-tools-intellij/liberty-tools-intellij/liberty-tools-intellij/src/test/resources/projects/gradle/singleModGradleMP/build.gradle. Could not resolve build file.
java.net.MalformedURLException: no protocol: /home/runner/work/liberty-tools-intellij/liberty-tools-intellij/liberty-tools-intellij/src/test/resources/projects/gradle/singleModGradleMP/build.gradle
	at java.base/java.net.URL.<init>(URL.java:674)
	at java.base/java.net.URL.<init>(URL.java:569)
	at java.base/java.net.URL.<init>(URL.java:516)
	at io.openliberty.tools.intellij.runConfiguration.LibertyRunManagerListener.runConfigurationRemoved(LibertyRunManagerListener.java:42)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:699)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:659)
	at com.intellij.util.messages.impl.MessageBusImplKt.deliverMessage(MessageBusImpl.kt:422)
	at com.intellij.util.messages.impl.MessageBusImplKt.pumpWaiting(MessageBusImpl.kt:401)
	at com.intellij.util.messages.impl.MessageBusImplKt.access$pumpWaiting(MessageBusImpl.kt:1)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:460)
	at jdk.proxy2/jdk.proxy2.$Proxy106.runConfigurationRemoved(Unknown Source)
	at
```

Also as part of this issue, write a unit or integration test to cover this scenario.

**Comment by anusreelakshmi934** [comment]
The problem stemmed from an incorrect path to the build file, leading to an exception. After resolving this issue and writing the corresponding test, we encountered differing outcomes during PR builds: failure on Windows, success on Linux, and a known failure on Mac due to a specific issue. The test `testCustomStartParametersClearedOnConfigRemoval` passed on Mac's Gradle but encountered an initialization error on Maven.

To diagnose the problem, I ran the tests locally. Setting up a Windows VM, I executed the tests, and they passed without any issues, resulting in success. But it seem to fail in PR build.
